templates/user/my_donation.html.twig

```diff
@@ -24,28 +24,30 @@
         {% endif %}
     </div>
 
-    <table class="settings__donations">
-        <thead>
-            <tr>
-                <th>
-                    Date
-                </th>
-                <th>
-                    Montant
-                </th>
-            </tr>
-        </thead>
-        <tbody>
-            {% for donation in donations|reverse %}
+    {% if donations %}
+        <table class="settings__donations">
+            <thead>
                 <tr>
-                    <td>
-                        {{ donation.donatedAt|date('d M Y') }}
-                    </td>
-                    <td>
-                        {{ donation.amountInEuros }} €
-                    </td>
+                    <th>
+                        Date
+                    </th>
+                    <th>
+                        Montant
+                    </th>
                 </tr>
-            {% endfor %}
-        </tbody>
-    </table>
+            </thead>
+            <tbody>
+                {% for donation in donations|reverse %}
+                    <tr>
+                        <td>
+                            {{ donation.donatedAt|date('d M Y') }}
+                        </td>
+                        <td>
+                            {{ donation.amountInEuros }} €
+                        </td>
+                    </tr>
```

Comment:
you can do this with `{% else %}` and add one message:
```
{% for donation in donations|reverse %}
...
{% else %}
    <tr><td colspan="2">Vous n'avez encore fait de don</td></tr>
{% endfor %}
```

Reply:
Good point. Thank you! Will change.

Reply:
Actually, that's not the only area that needs to disappear, I made another tweak instead.